.github/ISSUE_TEMPLATE/marketing-request.yml

```diff
@@ -3,9 +3,42 @@ description: Get support on sending a message out to our community through coord
 title: 'REQUEST: New communication about <topic>'
 labels: area/contributor-comms, sig/contributor-experience
 body:
-- type: textarea
-  attributes:
-    label: What do you want send out?(Please include a link to a draft)
-    description: Gists, Hackmd.io, or Google Doc preferred. Include any artwork, pics, diagrams, or any other assets with public links. We use a combination of mailing lists, Slack messages, and Twitter to get the word out.
-  validations:
-    required: true
\ No newline at end of file
+  - type: textarea
+    id: content
+    attributes:
+      label: What do you want send out?(Please include a link to any information or a draft)
+      description: Gists, Hackmd.io, or Google Doc preferred. Include any artwork, pics, diagrams, or any other assets with public links.
+    validations:
+      required: true
+  - type: dropdown
+    id: help
+    attributes:
+      label: Drafting Content help needed?
+      description: Do you also need help drafting the content?
+      options:
+        - "Yes"
+        - "No"
+      default: 0
+    validations:
+      required: true
+  - type: dropdown
+    id: audience
+    attributes:
+      label: Target Audience
+      description: Contributor Comms have different Channels that aim at different target audiences. To determain what channels to utilize please state your audience for the request.
```

Comment:
```suggestion
      description: Contributor Comms has different channels that aim at different target audiences. To determine what channels to utilize, please state your audience for the request.
```

Reply:
So, we have K8sContributors (for the community) or Kubernetesio (for all humanity) so it might be another drop down. Something like "Contributors/Maintainers" and "End Users". Then we add more personas as accounts.